Gemfile.lock

```diff
@@ -78,29 +78,29 @@ GEM
       public_suffix (>= 2.0.2, < 5.0)
     airbrussh (1.4.0)
       sshkit (>= 1.6.1, != 1.7.0)
-    annotate (3.0.3)
+    annotate (3.1.1)
       activerecord (>= 3.2, < 7.0)
       rake (>= 10.4, < 14.0)
-    apipie-rails (0.5.16)
+    apipie-rails (0.5.17)
       rails (>= 4.1)
     arbre (1.2.1)
       activesupport (>= 3.0.0)
     ast (2.4.0)
     aws-eventstream (1.0.3)
-    aws-partitions (1.258.0)
-    aws-sdk-core (3.86.0)
+    aws-partitions (1.289.0)
+    aws-sdk-core (3.92.0)
       aws-eventstream (~> 1.0, >= 1.0.2)
       aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.27.0)
+    aws-sdk-kms (1.30.0)
       aws-sdk-core (~> 3, >= 3.71.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.60.1)
+    aws-sdk-s3 (1.61.1)
       aws-sdk-core (~> 3, >= 3.83.0)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.1.0)
+    aws-sigv4 (1.1.1)
```

Comment:
### [aws-sigv4](https://github.com/aws/aws-sdk-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.0 → 1.1.1

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sigv4/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sigv4)_]





<details>
<summary>Commits</summary>

A change of **1** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/1.1.0...1.1.1).

These are the individual commits:
- (cfd13e3) [Version 1.1.1 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/cfd13e3718110808e3074d5a2f96c8d01815d52d)


</details>